lib/cib/cib_utils.c

```diff
@@ -650,11 +650,6 @@ cib_internal_op(cib_t * cib, const char *op, const char *host,
                      const char *section, xmlNode * data,
                      xmlNode ** output_data, int call_options, const char *user_name) =
         cib->delegate_fn;
-
-    if(user_name == NULL) {
-        user_name = getenv("CIB_user");
-    }
-
```

Comment:
We don't need to remove this code -- just unset CIB_user before you run the tests. Normally CIB_user is not defined, but it's used for remote CIB administration ( https://clusterlabs.org/pacemaker/doc/2.1/Pacemaker_Administration/singlehtml/index.html#connecting-from-a-remote-machine )